packages/deploy/test/fixtures.ts

```diff
@@ -75,14 +76,18 @@ export function fullExampleState() {
         edges: {
           'trigger-one->job-a': {
             id: 'ea264d6d-9767-4a2c-810f-deb10961a6dc',
-            condition: null,
+            condition_type: null,
```

Comment:
i'd expect this to be `always`

Reply:
@taylordowns2000 The pre-existing field `condition` (now renamed to `condition_type`) is a nullable column. May I apply a migration on Lightning converting all `NULL` to `'always'`? To be more conservative, we can update only where `source_trigger_id is not NULL`.